spec/unit/food_truck_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../../app/food_truck'
+require_relative '../../constants'
+
+module FoodTruck
+  RSpec.describe FoodTruck do
+    describe 'FoodTruck loads data' do
```

Comment:
Nesting like this is fine -- it does not cause any harm -- but usually it is done for a reason, and it's not clear to me what you are trying to achieve by nesting it.  This could just be:

``` ruby
module FoodTruck
  RSpec.describe FoodTruck do
    it 'provides food truck names' do
      # ...
    end
  end
end
```

...and it would work just fine.  Your nesting isn't making it behave any different other than changing your spec's full description from the readable `FoodTruck provides food truck names` to the unreadable `FoodTruck FoodTruck loads data provides food truck names`.